io.catenax.pcf/4.0.0/Pcf.ttl

```diff
@@ -0,0 +1,896 @@
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:properties ( :productId ) .
+
+:declaredUnit a samm:Property ;
+   samm:preferredName "Unit of measurement"@en ;
+   samm:description "Unit of analysis of a product in context of the PCF (product carbon footprint) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. In Catena-X list of valid units includes \"piece\"."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :DeclaredUnitCharacteristic ;
+   samm:exampleValue "kilogram" .
+
+:unitaryProductAmount a samm:Property ;
+   samm:preferredName "Unitary Product Amount"@en ;
+   samm:description "Amount of units contained within a product in context of the PCF (Product Carbon Footprint) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :StrictlyPositiveDecimalTrait ;
+   samm:exampleValue "1000.0"^^xsd:float .
+
+:productMassPerDeclaredUnit a samm:Property ;
+   samm:preferredName "Product Mass Per Declared Unit"@en ;
+   samm:description "Mass of a product per declared unit in context of the PCF (Product Carbon Footprint) as specified in the Catena-X PCF Rulebook."@en ;
+   samm:characteristic :PositiveDecimalWeightTrait ;
+   samm:exampleValue "0.456"^^xsd:decimal .
+
+:exemptedEmissionsPercent a samm:Property ;
+   samm:preferredName "Exempted Emissions Percent"@en ;
+   samm:description "Percentage of emissions excluded from PCF (Product Carbon Footprint) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :ExemptedEmissionsPercentTrait ;
+   samm:exampleValue "0.0"^^xsd:decimal .
+
+:exemptedEmissionsDescription a samm:Property ;
+   samm:preferredName "Exempted Emissions Description"@en ;
+   samm:description "Rationale behind exclusion of specific PCF (Product Carbon Footprint) emissions as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "No exemption" .
```

Comment:
type info? also check the other elements

Reply:
Not sure what you mean. Type is standard string/ text. How should i include a "type info"? Could you provide an example?

Reply:
Additional ^^xsd:string type info seems to be irrelevant as not added by the model editor. Model validates nevertheless, so no need for modification.